### PR TITLE
update embeddings tool to generate updated embeddings.yaml structure

### PR DESCRIPTION
**Pull request by chejennifer**
Update build_custom_dc_embeddings to work with the new structure of the embeddings.yaml file in this PR: https://github.com/datacommonsorg/website/pull/4212

### CODE REVIEW

File: tools/nl/embeddings/build_custom_dc_embeddings.py [hunk 1]
```diff
@@ -95,9 +96,10 @@ def download(embeddings_yaml_path: str):
       embeddings_yaml_path, default_ft_embeddings_info)
 
 
-def build(model_version: str, sv_sentences_csv_path: str, output_dir: str):
```
Comment (keyurva) [inline tools/nl/embeddings/build_custom_dc_embeddings.py:98]: Can you run these 2 commands to ensure the script runs as expected:

```
build_custom_dc_embeddings.py --mode=download --embeddings_yaml_path=tmp/embeddings.yaml
```

Once this runs successfully, check the generated `tmp/embeddings.yaml` and validate it conforms to the new format.

> Login to gcloud before running the next one.

```
python3 build_custom_dc_embeddings.py \                                                      
--sv_sentences_csv_path=gs://datcom-floret-dev-resources/temp/nl/sentences.csv \
--output_dir=tmp
```

Check that it generates an embeddings CSV in the `tmp` folder.

> You may need to create a `tmp` folder before running these commands.

Comment (chejennifer) [reply tools/nl/embeddings/build_custom_dc_embeddings.py:98]: thanks for the pointer, scripts run as expected

outputs
/tmp/embeddings.yaml: https://paste.googleplex.com/6011620073668608
/tmp/custom_embeddings.yaml: https://paste.googleplex.com/4865867779407872

**Review by keyurva — APPROVED** [review]
Thanks for the updates!

File: tools/nl/embeddings/utils.py [hunk 2]
```diff
@@ -18,12 +18,10 @@
...
 import os
 from pathlib import Path
-import re
 from typing import Any, Dict, List, Tuple
 
 from file_util import create_file_handler
 from google.cloud import aiplatform
-from google.cloud import storage
 import pandas as pd
 from sentence_transformers import SentenceTransformer
```
Comment (pradh) [inline tools/nl/embeddings/utils.py:26]: Is this used?

File: tools/nl/embeddings/utils.py [hunk 3]
```diff
@@ -55,10 +53,25 @@ def _is_gcs_path(path: str) -> bool:
   return path.strip().startswith(_GCS_PATH_PREFIX)
 
 
-def _get_gcs_parts(gcs_path: str) -> tuple[str, str]:
+def _get_gcs_parts(gcs_path: str) -> Tuple[str, str]:
   return gcs_path[len(_GCS_PATH_PREFIX):].split('/', 1)
 
 
+@dataclass
+class ModelInfo:
```
Comment (pradh) [inline tools/nl/embeddings/utils.py:61]: Renamed to Config now?

**Review by pradh — APPROVED** [review]

File: tools/nl/embeddings/utils.py (hunk 2 above)
Comment (chejennifer) [reply tools/nl/embeddings/utils.py:26]: SentenceTransformer? yes

File: tools/nl/embeddings/utils.py (hunk 3 above)
Comment (chejennifer) [reply tools/nl/embeddings/utils.py]: this is just specific to the embedding tool but updated to make it consistent

**Review by pradh — APPROVED** [review]
Thanks

**Pull request merged**